### DISCUSSION TITLE
Created a User and deleted and when trying to add the same user email ID "User Already" taken

### DISCUSSION THREAD

**Discussion by ram-srikhakollu** [community/community#34581] [New to GitHub]
Created a User and deleted and when trying to add the same user email ID "User Already" taken is shown.

**Comment by jfullstackdev** [comment]
you mean, you created a user, add an email, then deleted it afterwards

then, you made another user or an existing one, then you added the email, it says, "it's already taken" ?

yes, as far as I know, it will be 90 days or never at all, there are issues regarding this, like releasing of usernames

maybe try the Support although there is no assurance

https://support.github.com/contact/account
